Section 3.1.3 (Operational Governance) copyedits (#31)
rfc-editor/draft-nottingham-avoiding-internet-centralization: draft-nottingham-avoiding-internet-centralization.md

### Operational Governance {#multi}

Federation and distributed consensus can both create the conditions for the provision of a function by multiple providers, but they cannot guarantee it. However, when providers require access to a resource or cooperation of others to provide that service, that choke point can itself be used to influence provider behavior -- including in ways that can counteract centralization.

> In these circumstances, some form of governance over that choke point is necessary to assure the desired outcome. Often, this is through the establishment of a multi-stakeholder body, i.e., an institution that includes representatives of the different kinds of parties that are affected by the system's operation ("stakeholders") in an attempt to make well-reasoned, legitimate, and authoritative decisions.

Reviewer: "i.e." is a bit awkward here, can we try a different construction?

Reviewer 2: Is this update acceptable?

```suggestion
In these circumstances, some form of governance over that choke point is necessary to assure the desired outcome. Often, this is through the establishment of a multi-stakeholder body, which is an institution that includes representatives of the different kinds of parties that are affected by the system's operation ("stakeholders") in an attempt to make well-reasoned, legitimate, and authoritative decisions.
```

Reviewer: That's better, thanks.

Reviewer 2: This update has been implemented. @mnot, please review.

The most widely studied example of this technique is the governance of the DNS name space, which exhibits centralization as a “single source of truth”. That source of truth is overseen by [the Internet Corporation for Assigned Names and Numbers (ICANN)](https://www.icann.org/resources/pages/governance/governance-en), a global multi-stakeholder body with representation from end users, governments, operators, and others.

> Another example is the governance of the Web's trust model, implemented by web browsers as relying parties that have strong incentives to protect user privacy and security and CAs as trust anchors that have a strong incentive to be included in browser trust stores. To promote the operational and security requirements necessary to provide the desired properties, the [CA/Browser Forum](https://cabforum.org) was established as an oversight body that involves both parties as stakeholders.

Reviewer: Just checking, has CA been expanded above?

Reviewer 2: The acronym was added to the first occurrence of the expanded form; see #23.
